drivers/serial/uart_nrfx_uarte.c

```diff
@@ -1983,7 +1983,7 @@ static int uarte_nrfx_pm_action(const struct device *dev,
 #define UART_NRF_UARTE_DEVICE(idx)					       \
 	UARTE_INT_DRIVEN(idx);						       \
 	UARTE_ASYNC(idx);						       \
-	IF_ENABLED(CONFIG_PINCTRL, (PINCTRL_DT_DEFINE(UARTE(idx))))	       \
+	IF_ENABLED(CONFIG_PINCTRL, (PINCTRL_DT_DEFINE(UARTE(idx))));	       \
```

Comment:
For consistency, this should be actually:
```suggestion
	IF_ENABLED(CONFIG_PINCTRL, (PINCTRL_DT_DEFINE(UARTE(idx));))	       \
```
but even with this extra semicolon for !PINCTRL, it is acceptable.

Reply:
thanks, adjusted